### ISSUE TITLE
Can't cancel alarm creation

### ISSUE THREAD

**Issue by ChristopherRogers1991** [MycroftAI/skill-alarm#15]
I was trying to test the Mark 1 enclosure skill (it's stealing my brightness intents) and mycroft tried to set an alarm for me. Once he started asking for am/pm, I couldn't get out without giving one of those as an answer.

Iaw this on a Mark 1 first, and then used https://github.com/ChristopherRogers1991/mycroft-chatter on my desktop to generate the chat log below:
```
Me: set 50%
Mycroft: Should I set this for A.M. or P.M.?
Me: stop
Mycroft: Should I set this for A.M. or P.M.?
Me: cancel
Mycroft: Should I set this for A.M. or P.M.?
Me: am
Mycroft: Okay. Setting a 50% a.m. alarm
Mycroft: 50% a.m. alarm is up
Me: cancel
Mycroft: I'm sorry, I don't understand.
Me: cancel my alarm
Mycroft: I can not find an alarm set for 
Me: cancel my alarm at 50%
Mycroft: I can not find an alarm set for 
Me: list my alarms
Mycroft: I don't understand, but I'm learning new things everyday.
Me: when is my next alarm
Mycroft: It seems like you have no active alarms
```

**Comment by penrods** [comment]
Should be resolved by #26 once it is merged into the Mycroft Skills repo

**Comment by penrods** [comment]
Should be working now
